product_docs/docs/migration_toolkit/55/05_installing_mtk/index.mdx

```diff
@@ -28,8 +28,14 @@ You can install Migration Toolkit on:
 
 <div id="installing_drivers" class="registered_link"></div>
 
-## Installing source-specific drivers
+## Installing EDB JDBC drivers
 
-Before invoking Migration Toolkit, you must download and install a freely available source-specific driver. To download a driver, or for a link to a vendor download site, see [Third Party JDBC Drivers](https://www.enterprisedb.com/software-downloads-postgres#third-party-jdbc-drivers) on the Downloads page.
+If your source or target will be EDB Postgres Advanced Server, you must download and install a JDBC driver provided by EDB before invoking Migration Toolkit.  To download the latest driver, see [EDB Connectors](https://enterprisedb.com/software-downloads-postgres#edb-connectors) on the Downloads page.
+
+After downloading the EDB driver, move the driver file into the `<mtk_install_dir>/lib` directory.
+
+## Installing third-party JDBC drivers
+
+If your source will be Oracle, Microsoft SQL Server or MySQL, you must download and install a freely available source-specific driver before invoking Migration Toolkit. For a link to a vendor download site, see [Third Party JDBC Drivers](https://www.enterprisedb.com/software-downloads-postgres#third-party-jdbc-drivers) on the Downloads page.
```

Comment:
What is on the Downloads page is a link to the interactive installer not a link to the vendor download site.

Reply:
Should we just circumvent the mess and have the links to the vendor sites in the doc?

Reply:
I think having it in the docs (in addition to the website) makes sense, although we would need to make sure the documentation is updated in case the location of the download pages change.  Of course, that's no different than what we have to do for the website.  Do you need me to provide the links to include them in the documentation?

Reply:
Done